README.Rmd

```diff
@@ -63,83 +63,80 @@ devtools::install_github("bcgov/bcgovr")
 
 ### Usage
 
-#### analysis_skeleton()
-Most first time users will want to start with the `analysis_skeleton()` function. If RStudio is still open from the above [Installation] steps, you can now immediately auto-populate a new R-based open source data analysis project folder, including all of the [required bcgov items](https://github.com/bcgov/BC-Policy-Framework-For-GitHub/blob/master/BC-Gov-Org-HowTo/Cheatsheet.md), with `analysis_skeleton`. You can use `bcgovr::analysis_skeleton()` which directly calls the function from `bcgovr` library rather than loading the entire library. By default `analysis_skeleton()` will create a 'ready-to-go' directory in the current working directory. You can check the working directory using `getwd()`. The project name will be inherited from the working directory:
+#### analysis_skeleton
+
+The analysis_skeleton function auto-populates a new R-based open source data analysis project with folders & files that encourage best practice in scientific computing and including all of the [required bcgov items](https://github.com/bcgov/BC-Policy-Framework-For-GitHub/blob/master/BC-Gov-Org-HowTo/Cheatsheet.md).
+
+**Step 1: Set up a [remote repository](https://help.github.com/articles/about-remote-repositories/)**
+
+For a remote in [github.com/bcgov](github.com/bcgov), click on the green **New** button to create a new repository. [Choose a repository name](https://github.com/bcgov/BC-Policy-Framework-For-GitHub/blob/master/BC-Gov-Org-HowTo/Naming-Repos.md)---our example repository name is `bcgovr_analysis`. You can open an empty repository---without initializing a README, a .gitignore file or a license--- as `bcgovr` will take care of all of that for you later. Copy the URL of the repository, you'll need it to set up your local repository in the next step.
+
+
+**Step 2: Set up & populate local repository using `bcgovr`**
+
+*RStudio GUI users:* Open a fresh session of RStudio where you will open your New Project. By default, RStudio will create any New Project in the current working directory. You can check the working directory using `getwd()`. Set your preferred local working directory in the R console (e.g. `setwd(C:/_dev)`) or using the RStudio navigation tools (Session -> Set Working Directory -> Choose Directory...). Now open your New Project: Select Version Control from the Create Project menu -> Select Git -> Paste the URL of the github.com/bcgov repository from step 1 in Repository URL -> Click 'Create Project'. The local project name will be inherited from the remote repository, and your local project is now "connected" to your remote in [github.com/bcgov](github.com/bcgov).
```

Comment:
I'm not sure if we need the discussion on setting a working directory, as when you use the new project it will set it for you... I think?